app/main.py

```diff
@@ -1,8 +1,58 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List[Car]) -> float:
+        total_cost = 0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_cost += CarWashStation.calculate_washing_price(self, car)
+                CarWashStation.wash_single_car(self, car)
+        return total_cost
+
+    def calculate_washing_price(self, car: Car) -> float:
+        washing_cost = round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center
+            , 1)
+        return washing_cost
+
+    def wash_single_car(self, car: Car) -> Car:
+        car.clean_mark = self.clean_power
+        return car
```

Comment:
<img width="603" alt="image" src="https://github.com/mate-academy/py-car-wash-station/assets/93856423/f8644a61-7345-496e-9f01-100e4b5ceefb">

Reply:
OK. But after all, then we will get a repetition of code, because the cars already sorted by the given condition in the another method ("serve_cars")